serverless.yml

```diff
@@ -0,0 +1,39 @@
+service: isomercms
+
+frameworkVersion: "3"
+
+plugins:
+  # serverless-plugin-typescript needs to precede serverless-offline
+  - serverless-plugin-typescript
+  - serverless-step-functions
+  - serverless-step-functions-local
+  - serverless-offline-lambda
+  - serverless-offline
+
+provider:
+  name: aws
+  runtime: nodejs14.x
+
+custom:
+  stepFunctionsLocal:
+    accountId: 101010101010
+    region: us-east-1
+    lambdaEndpoint: http://localhost:3002
+    TaskResourceMapping:
+      Hello: arn:aws:lambda:us-east-1:101010101010:function:hello
+      Hithere: arn:aws:lambda:us-east-1:101010101010:function:hithere
```

Comment:
I'm probably not understanding this but does this need to match the one in the `step-functions` folder? The region is different.

Also, are we able to reuse the definitions in line 19 and 20? Something similar to `${self:accountId}`, not sure if this is possible?

Reply:
Hey - these are 'fake' functions that were created. `Hello` and `Hithere` were just functions we created to test the functionality of local stack. These are to be removed in the later commits using the variables that you have described!